Documentation/CMake.md

```diff
@@ -59,21 +59,28 @@ endif()
 ## Add an entry point
 
 You must include a source file in your test executable target with a
-`@main` entry point. The following example uses the SwiftPM entry point:
+`@main` entry point.
 
 ```swift
-import Testing
+import Foundation
+@_spi(ForToolsIntegrationOnly) import Testing
```

Comment:
So, yes-and-no. As we talked about in Slack, it's a bug that any `@_spi` content is available in the Swift toolchain right now and it's going to get removed in the future.

For the moment, code should either use `dlsym()/`GetProcAddress()` or, if a little more adventurous, use the experimental `@_extern(c)` to call `swt_abiv0_getEntryPoint()`, then load the SwiftCC entry point from there. (`@_silgen_name` could be used too, but technically the calling conventions are incompatible, so _caveat codor_.)

We're still looking at how to expose these symbols in a way that makes it easy for test tools/harness authors but doesn't make the symbols visible to test authors by default. We're looking at adding a dedicated target in the package (if we can do so without requiring building the whole library from source) or creating a separate `swift-testing-tools-support` package, among other options.

Reply:
I've replaced the spi call with an explicit extern(c) for the symbol. I mentioned in a note above the example code that you could do the same with a C header file and a module.modulemap to fwd declare it.